### ISSUE TITLE
FYI - Launching scoop installed Ludusavi with default Terminal set to Windows Terminal strips environment variables

### ISSUE THREAD

**Issue by sharkusmanch** [mtkennerly/ludusavi#207]
### Ludusavi version

v0.18.0

### Operating system

Windows

### Installation method

Standalone

### Description

**Not really a ludusavi issue, but more of a Windows/scoop shortcut issue. But wanted to share nonetheless in case someone else hits this.**

### TLDR;

When windows default terminal is set to `Windows Terminal` (or in my case `Let Windows decide`) and using the scoop created Ludusavi Start Menu shortcut, environment variable are not passed to the Ludusavi GUI.

If you're setting rclone environment variables for your user profile, and the above is true, the cloud sync will likely fail with error: `The request is not supported. (os error 50)`

CLI transfers will still work fine, only GUI is impacted.

### Solution
 
Change default Windows Terminal to be `Windows Console Host`

![terminal](https://github.com/mtkennerly/ludusavi/assets/9328629/96942e7a-3df1-420d-8a70-e24e3409ccc6)

After changing that setting and relaunching Ludusavi via the Start Menu shortcut, cloud transfers work fine in my case.

### More context:

- Rclone config is encrypted. Password command to decrypt is specified via ENV. Solution described [here ](https://forum.rclone.org/t/how-to-use-rclone-password-command-with-windows-powershell-for-config-password/15950)
- Ludusavi and rclone installed via scoop
- GUI transfers were failing for me, but CLI transfers were not having any issues.
- Noticed launching Ludusavi via start menu shortcut was also launching a minimized Windows Terminal process
- Switching default Terminal app to `Windows Console Host` fixed the GUI issue for me

rclone environment variables set for my user profile:
```
PS C:\Users\marcus\Sync\Config> $env:RCLONE_CONFIG
C:\Users\marcus\Sync\Config\rclone.conf
PS C:\Users\marcus\Sync\Config> $env:RCLONE_PASSWORD_COMMAND
powershell C:\Users\marcus\Sync\Config\rclone_pass.ps1

```
rclone backend:
```
PS C:\Users\marcus> rclone config show ludusavi -vv
2023/05/20 19:48:28 DEBUG : Setting --config "C:\\Users\\marcus\\Sync\\Config\\rclone.conf" from environment variable RCLONE_CONFIG="C:\\Users\\marcus\\Sync\\Config\\rclone.conf"
2023/05/20 19:48:28 DEBUG : Setting --password-command "powershell C:\\Users\\marcus\\Sync\\Config\\rclone_pass.ps1" from environment variable RCLONE_PASSWORD_COMMAND="powershell C:\\Users\\marcus\\Sync\\Config\\rclone_pass.ps1"
2023/05/20 19:48:28 DEBUG : rclone: Version "v1.62.2" starting with parameters ["C:\\Users\\marcus\\scoop\\apps\\rclone\\current\\rclone.exe" "config" "show" "ludusavi" "-vv"]
[ludusavi]
2023/05/20 19:48:28 DEBUG : Using config file from "C:\\Users\\marcus\\Sync\\Config\\rclone.conf"
type = alias
remote = pcloud-crypt:/ludusavi
2023/05/20 19:48:28 DEBUG : rclone: Version "v1.62.2" finishing with parameters ["C:\\Users\\marcus\\scoop\\apps\\rclone\\current\\rclone.exe" "config" "show" "ludusavi" "-vv"]
```

**Comment by sharkusmanch** [comment]
Going to close this, as I mentioned it's not really a ludusavi issue. Just wanted to share something searchable for others that might hit this issue.

**Comment by mtkennerly** [comment]
I wonder if there's a way to mitigate this in Ludusavi. However, I can't reproduce it more generally. Here's what I tried:

* Made a build of Ludusavi that logs some environment variables on startup (some at the user level and some at the system level). It also runs a Python script that does the same logging, to see if the variables make it through an extra layer.
* Installed Ludusavi via Scoop.
* Replaced `%USERPROFILE%\scoop\apps\ludusavi\0.18.0\ludusavi.exe` with the custom build.
* Ran Ludusavi from the Start Menu and from `%USERPROFILE%\scoop\shims\ludusavi.exe`.

I haven't tried specifically with the encrypted Rclone config, but I'm not sure what would be special about that compared to any other environment variable 🤔 

Environment:

* Windows 11
* Windows Terminal 1.16.10262.0
* Scoop 0.3.1

**Comment by mtkennerly** [comment]
Ah, I was just able to reproduce this, but it isn't because of the environment variables. It seems to be related to how Ludusavi uses `FreeConsole` to hide the console in GUI mode. The issue happens if you launch the exe from Windows Explorer, but not if you launch it from a console. I always test with `cargo run`, so that explains why I didn't run into this before.

I'm not sure if this is technically a bug in Ludusavi or in Windows Terminal, but it's not the first issue I've had related to `FreeConsole`, so I think I'm just going to change the implementation to avoid using it.

**Comment by mtkennerly** [comment]
This should be fixed now in v0.18.1.
